src/components/OS/Snippet/Snippet.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { styled, IllustratedSection } from '@aragon/ui'
+import OverlapBackground from '../../OverlapBackground/OverlapBackground'
+
+import code from './assets/code.png'
```

Comment:
I would use a more serious app such as the voting app for the code example :)

Reply:
Tried to skim through it but I didn't find any snippet that would fit. Maybe we can have the contract is AragonApp thing, then cut everything else and display the roles, and show a function behind the ACL. What do you think?

Reply:
hmmm i guess it is fine

Reply:
LGTM